frontend/src/views/File.vue

```diff
@@ -199,6 +199,10 @@ main#filelanding
             </dd>
             <dt>Level</dt>
             <dd>{{ response.product.level }}</dd>
+            <dt v-if="response.instrumentPid">Instrument</dt>
+            <dd v-if="response.instrumentPid">
+              <a :href="response.instrumentPid">{{ instrument }} </a>
+            </dd>
```

Comment:
Voi halutessa vähentää toistoa `<template>`:lla:

```suggestion
            <template v-if="response.instrumentPid">
              <dt>Instrument</dt>
              <dd><a :href="response.instrumentPid">{{ instrument }}</a></dd>
            </template>
```